docker-compose.yml

```diff
@@ -89,7 +89,7 @@ services:
       DATABASE_URL: postgresql://forem:forem@db:5432/PracticalDeveloper_development
     volumes:
       - .:/opt/apps/forem:delegated
-    entrypoint: ["dockerize", "-wait", "tcp://db:5432", "-wait", "tcp://redis:6379", "-wait", "http://rails:3000", "-timeout", "2700s", "-wait-retry-interval", "20s"]
+    entrypoint: ["dockerize", "-wait", "tcp://db:5432", "-wait", "tcp://redis:6379", "-wait", "tcp://rails:3000", "-timeout", "2700s", "-wait-retry-interval", "20s"]
```

Comment:
`tcp://host:port` only checks that a service is _listening_, not that it's functioning. I _think_ that's okay in this case (we really are waiting on the webserver to confirm all of the things _it_ waits for, like bundle and db:prepare, have completed, rather than needing the webserver live in most cases to start the seed or background jobs.

I played around a little with the `-wait-http-header ` argument to see if I could change "Host" but didn't get the expected response (this still gives a 403 and may be ignoring the host header or have already sent one). 

Digging through how dockerize handles http schema, I think this test case disproves being able to modify the host from the url:

```go
package main

import (
	"net/http"
)

func main() {
	client := &http.Client{}
	req, _ := http.NewRequest("GET", "http://localhost:4000", nil)
	req.Header.Add("host", "hostheader")
	req.Header.Add("nonhost", "thenonhostheader")
	client.Do(req)
}
```

Now wait for it to execute by starting a tcp service - I do not get back the requested host header, suggesting Go doesn't support overriding the url host by passing a host header (the added header is just dropped):

```
$ nc -l 4000
GET / HTTP/1.1
Host: localhost:4000
User-Agent: Go-http-client/1.1
Nonhost: thenonhostheader
Accept-Encoding: gzip

```